### PR TITLE
react-lists - Skylar Shaffer

### PR DESCRIPTION
**Pull request by skylarshaffer**
Completed react-lists challenge without dynamic creation. Working now on adding that.
image

### CODE REVIEW

File: react-lists/my-app/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' },
+];
+
+export default function App() {
+  return (
+    <>
```
Comment (RobertGardner) [inline react-lists/my-app/src/App.tsx:19]: The fragment is unnecessary.

File: react-lists/my-app/src/PokemonList.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  number: string;
+  name: string;
+};
+
+type Props = {
+  pokedex: Pokemon[];
+};
+
+export function PokemonList({ pokedex }: Props) {
+  return (
```
Comment (RobertGardner) [inline react-lists/my-app/src/PokemonList.tsx:11]: This works for this exercise, but it's not general. What if the list had 0 element? Or 10?

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Sorry, you need to follow the technique in the docs so it will work with any number of elements.

File: react-lists/my-app/src/PokemonList.tsx (hunk 2 above)
Comment (skylarshaffer) [reply react-lists/my-app/src/PokemonList.tsx]: Temporary measure for quicker submission. Has been resolved.

File: react-lists/my-app/src/App.tsx (hunk 1 above)
Comment (skylarshaffer) [reply react-lists/my-app/src/App.tsx]: Resolved.

**Comment by skylarshaffer** [comment]
> Sorry, you need to follow the technique in the docs so it will work with any number of elements.

Discussed, as this was simply an incomplete submission for the purpose of speed. Challenge has been improved and is ready for re-review.

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**